src/pages/[platform]/build-a-backend/storage/storagepath/index.mdx

```diff
@@ -0,0 +1,125 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Using StoragePath',
+  description: "Learn more about constructing a StoragePath to use on Amplify Storage APIs",
+  platforms: [
+    'swift',
+    'android',
+    'flutter'
+  ]
+};
```

Comment:
I'd rework the title here a bit. "Using StoragePath" doesn't say much. @hdworld11 maybe you have some ideas

Reply:
"Using new StoragePath parameter"?

Reply:
I guess my question is, does it warrant its own page? Isn't this just part of the list/upload APIs?

Reply:
We thought about including it in the individual API pages, but I think the dedicated page makes sense because we want to be extra clear to the customer on how to use this field.

And since this is in Gen 1, we want to be clear to customers on how they can achieve `key`/`accessLevel` behavior with `path` now.

React docs - https://docs.amplify.aws/gen1/react/build-a-backend/storage/path/